db/schema.rb

```diff
@@ -17,77 +16,76 @@
   enable_extension "plpgsql"
 
   create_table "assignments", force: :cascade do |t|
-    t.string   "name",           null: false
-    t.jsonb    "metadata"
-    t.string   "workflow_id",    null: false
-    t.string   "subject_set_id", null: false
-    t.integer  "classroom_id",   null: false
-    t.datetime "created_at",     null: false
-    t.datetime "updated_at",     null: false
+    t.string "name", null: false
+    t.jsonb "metadata"
+    t.string "workflow_id", null: false
+    t.string "subject_set_id", null: false
+    t.bigint "classroom_id", null: false
```

Comment:
https://github.com/rails/rails/pull/26266

I've changed the migrations to reflect the fact that they were generated with Rails 4.2 (not 5) and regenerated schema.rb. There was no change.